app/controllers/declaration/DraftDeclarationController.scala

```diff
@@ -16,17 +16,32 @@
 
 package controllers.declaration
 
-import controllers.actions.AuthAction
-import javax.inject.Inject
+import config.AppConfig
+import connectors.CustomsDeclareExportsConnector
+import controllers.actions.{AuthAction, JourneyAction, VerifiedEmailAction}
+import models.requests.ExportsSessionKeys
 import play.api.i18n.I18nSupport
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 import views.html.declaration.draft_declaration_page
 
-class DraftDeclarationController @Inject()(authenticate: AuthAction, mcc: MessagesControllerComponents, draftDeclarationPage: draft_declaration_page)
-    extends FrontendController(mcc) with I18nSupport {
+import javax.inject.Inject
+
+class DraftDeclarationController @Inject()(
+  authenticate: AuthAction,
+  appConfig: AppConfig,
+  mcc: MessagesControllerComponents,
+  draftDeclarationPage: draft_declaration_page,
+  journeyType: JourneyAction,
+  customsDeclareExportsConnector: CustomsDeclareExportsConnector,
+  verifyEmail: VerifiedEmailAction
```

Comment:
These two classes are not used in this controller so don't need to be injected :slightly_smiling_face: